Sources/Sentry/Public/SentryFeedbackAPI.h

```diff
@@ -12,9 +12,10 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-NS_EXTENSION_UNAVAILABLE("Sentry User Feedback UI cannot be used from app extensions.")
 @interface SentryFeedbackAPI : NSObject
 
+SENTRY_NO_INIT
+
```

Comment:
`m`: We should still mark the methods `showWidget` and `hideWidget` as unavailable in app extensions if UIKit is not available (not always the case, e.g. share extensions can have an UI).